### PR TITLE
Update requirements and travis to use Py3

### PR DESCRIPTION
**Pull request by dukebody**
We were using a very outdated version of Lektor and Flask, not compatible with Python 3. Update those. Also update travis to use Python 3.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,21 +1,23 @@
+-f /home/dukebody/.cache/pip/wheels/
```
Comment (rberenguel) [inline requirements.txt]: Psst! @dukebody

File: requirements.txt [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 argh==0.26.1
 Babel==2.2.0
 cffi==1.5.2
 click==6.3
 cryptography==1.3.1
 enum34==1.1.2
```
Comment (rberenguel) [inline requirements.txt]: Don't know why but locally with 3.6.5 `enum34` fails to install. Weird! But does not seem to be needed anyway

File: requirements.txt [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
 itsdangerous==0.24
-Jinja2==2.8
-Lektor==1.2.1
+Jinja2==2.10
+Lektor==3.1.1
 MarkupSafe==0.23
 mistune==0.7.2
 ndg-httpsclient==0.4.0
 pathtools==0.1.2
+pkg-resources==0.0.0
```
Comment (rberenguel) [inline requirements.txt]: Can remove this

File: requirements.txt [hunk 4]
```diff
@@ -1,21 +1,23 @@
+-f /home/dukebody/.cache/pip/wheels/
 argh==0.26.1
 Babel==2.2.0
 cffi==1.5.2
 click==6.3
 cryptography==1.3.1
```
Comment (rberenguel) [inline requirements.txt]: On Mac OS this version of cryptography seems to fail to install (may be something of my local setup). 1.5.1 installs and is compatible with Lektor 3.1.1 (most recent crypto is 2.2.2 though!)

Comment (dukebody) [reply requirements.txt]: I'm in favor of updating to the latest version of all libraries that doesn't break anything, so will update to crypto 2.2.2 and other latest packages.

File: .travis.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
 language: python
-python: 2.7
-install: "pip install Lektor==1.2.1"
+python: 3.6
+install: "pip install Lektor==3.1.1"
```
Comment (dukebody) [inline .travis.yml]: This line actually will install different packages that what we have in `requirements.txt`, so we should use `pip install -r requirements.txt` instead to make local builds same as remote ones.

File: requirements.txt (hunk 2 above)
Comment (dukebody) [reply requirements.txt]: Yes, it shouldn't be needed with Python 3.6.

See https://pypi.org/project/enum34/1.1.6/.

I will remove it.

File: .travis.yml (hunk 5 above)
Comment (rberenguel) [reply .travis.yml]: ?

Comment (dukebody) [reply .travis.yml]: Here in travis we are installing `Lektor==3.1.1` with all dependencies, which probably are the latest version of many packages. If locally we do instead `pip install -r requirements.txt` we will install older version of these dependencies, hence the difference in the two environments.

File: requirements.txt (hunk 4 above)
Comment (rberenguel) [reply requirements.txt]: Anything later than 1.8.1 made Lektor barf, though

File: .travis.yml (hunk 5 above)
Comment (rberenguel) [reply .travis.yml]: Don’t be so sure, Lektor will fail with a later cryptography.

File: requirements.txt (hunk 4 above)
Comment (dukebody) [reply requirements.txt]: I've upgraded all packages to the version resulting of installing `pip install Lektor==3.3.1` in a clean python 3 environment. It doesn't barf at me when I do `lektor server`. What about yours? Do you still get errors with the latest versions?

Comment (rberenguel) [reply requirements.txt]: No, this works fine also for me (and I got crypto 2.2.2 working). Weird, SSL libraries always play funny on Mac OS.... :+1:

File: .travis.yml (hunk 5 above)
Comment (ifosch) [reply .travis.yml]: I will test this in my local

File: README.md [hunk 6]
```diff
@@ -1 +1,19 @@
-# pybcn.github.io
+# Running a local copy
+
+1. Fork the repository and clone it.
+
+2. Install the requirements in a virtualenv using Python 3. For example, using `virtualenvwrapper`:
+
+```
+mkvirtualenv pybcn-web --python=/usr/bin/python3
+pip install requirements.txt
```
Comment (ifosch) [inline README.md]: I am missing a `source activate pybcn-web` here.
Also, I think the `pip install...` should use `-r`

File: .travis.yml (hunk 5 above)
Comment (ifosch) [reply .travis.yml]: I see the site ok in my local, without any errors.

**Pull request merged**